hack/verify-govet.sh

```diff
@@ -20,12 +20,13 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
 # For help output
 ARGHELP=""
 if [[ "$#" -gt 0 ]]; then
-    ARGHELP="WHAT='$@'"
+    # shellcheck disable=2034 # Variables sourced in other scripts executed from the same shell
```

Comment:
I still feel like I need more explanation here.  Why is this necessary? What's using ARGHELP and why do we have to do it this way vs. some other way that doesn't trigger shellcheck?